File: app/src/main/java/eu/lynxit/sampleapp/MainActivity.java
package eu.lynxit.sampleapp;

import android.os.Bundle;

import org.gearvrf.GVRActivity;

public class MainActivity extends GVRActivity {
    private ApplicationMain main;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        main = new ApplicationMain(this);
        setMain(main, "gvr.xml");
    }
}

File: app/src/main/java/eu/lynxit/sampleapp/MainActivity.kt
package eu.lynxit.sampleapp

import android.os.Bundle

import org.gearvrf.GVRActivity

class MainActivity : GVRActivity() {
    private var main: ApplicationMain? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        main = ApplicationMain(this)
        setMain(main, "gvr.xml")
    }
}
